exercise1/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+    color: #aa7443;
+    background-color: #fcf7da;
+    font-family: Verdana, Arial, sans-serif;
+    font-size: 90%;
+}
+
+img {
+    float: inherit;
+}
+
+h1 {
+    font-size: 110%;
+    margin-bottom: 0px;
+}
+
+#per {
+    background-color: #ad8a67;
+    margin-right: 85%;
+}
+
+#contact{
+    padding-left: 0px;
+    list-style-type: none;
+    border-style:dotted;
+}
+
+#all {
+    padding-left: 8px;
+    list-style-position:outside;
+}
+
+#edu {
+    border-style:dotted;
+}
+
+#skills {
+    border-style:dotted;
```

Comment:
Space after `:`. We need coding style consistency in the whole file. Correct when applicable.